### ISSUE TITLE
Incorporate jats2spar.xsl

### ISSUE THREAD

**Issue by Klortho** [Klortho/eutils-org#6]
I imported it into the repo:  https://github.com/Klortho/eutilsrdf/blob/master/jats2spar.xsl.
It is from [From Markup to Linked Data: Mapping NISO JATS v1.0 to RDF using the SPAR (Semantic Publishing and Referencing) Ontologies](http://www.ncbi.nlm.nih.gov/books/NBK100491/), by David Shotton, Silvio Peroni, and Debbie Lapeyre.
I'd like to get it working with esummary and efetch from PMC, inasmuch as possible.

**Comment by Klortho** [comment]
Maybe also esummary and efetch from PubMed -- that might require a new XSLT, but the data output should jibe.

See also what PubChem is doing (Gang, feel free to elaborate).  See, for example, http://pubchem.ncbi.nlm.nih.gov/rest/rdf/reference/PMID10395478.

**Comment by flamingofugang** [comment]
Hi Chris,

I suggest to work on esummary first. I have looked through JATS2RDF mappings and found they currently just mapped article and journal metadata into RDF:
"The JATS Journal Publishing Tag Library Version 1.0 specification is large, containing 246 elements and 134 attributes. We chose to map the JATS metadata entities that describe an article (e.g. <journal-meta> for metadata about the journal in which the article was published, such as the name of the journal), and to leave aside (possibly for a later mapping exercise using DoCO, the Document Components Ontology) those entities describing the textual and graphical structure and content of the article (e.g. <title>, <body>, <fig>, <table>)."

I am wondering whether the document components (e.g. abstract text ) and MeSH headlines in efetch can be well mapped...

**Comment by Klortho** [comment]
Hi, Gang,
I've been working on #12 for the last couple of days.  I decided to try to port to a new platform first, because:
- jats2spar.xsl is written in XSLT 2.0, which the existing Perl CGI can't handle,
- I am not sure I should be serving this thing from my NCBI staff account, and
- MarkLogic Server is a much better fit for this kind of app, anyway.

So far, I have just got the MLS instance up and running.  When it comes time to porting the code itself, I will work in a separate git branch.

I agree that we should start applying jats2spar to the esummary outputs first (both pubmed and pmc).
